app/reducers/users.ts

```diff
@@ -18,7 +18,7 @@ export type UserEntity = {
   profilePicture: string;
   profilePicturePlaceholder?: string;
   emailListsEnabled?: boolean;
-  selectedTheme: string;
+  selectedTheme?: string;
   photoConsents?: Array<PhotoConsent>;
```

Comment:
What's the point of keeping the `UserEntity` type when we have a `User` model in _app/models.ts_? Are they not the same? It seems like the only difference is `selectedTheme` and `emailListEnabled`.

Reply:
Yeah, the whole double typing thing is ridiculous. But I'm trying to move more and more types into the `app/models`. The `***Entity` types seem to be more for creating new entities and typing the actions, so we'll keep them for now.

In some of the changes here, you can see I replace a bunch of the `Entity` types with better types from `app/models` and I think that's the way to go here.